spec/models/merchant_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+# RSpec.describe Merchant, type: :model do
```

Comment:
Can we remove these comments and add shoulda matchers after doing the validations on the model?